apis/extension/preemption.go

```diff
@@ -0,0 +1,46 @@
+/*
+Copyright 2022 The Koordinator Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package extension
+
+import corev1 "k8s.io/api/core/v1"
+
+const (
+	// LabelPodPreemptionPolicy is used to set the PreemptionPolicy of a Pod when the feature
+	// PreemptionPolicyTransformer is enabled.
+	LabelPodPreemptionPolicy = SchedulingDomainPrefix + "/preemption-policy"
+	// LabelPodSetPreemptionPolicy determines whether the pod should set the DefaultPreemptionPolicy override
+	// the pod.spec.preemptionPolicy when the feature PreemptionPolicyTransformer enabled and the pod does not set
+	// the LabelPodPreemptionPolicy.
+	// Default use true when this label is not set.
+	LabelPodSetPreemptionPolicy = SchedulingDomainPrefix + "/set-preemption-policy"
```

Comment:
As discussed, this switch is not actually needed. When we need to remove the transformer logic, we can just make a global configuration change for the scheduler.

Reply:
fixed